### DISCUSSION TITLE
External Electric Fields

### DISCUSSION THREAD

**Discussion by jacekkozuch** [TinkerTools/tinker-gpu#210] [Ideas]
Dear all,

I have found this publication
https://aip.scitation.org/doi/full/10.1063/1.4861893
where the authors performed MD using the AMOEBA ff and Tinker in external fields. I haven't found any keywords in the manual (maybe I've missed something) and the authors say that they used a modified version of Tinker. Therefore I assume that external electric fields are not implemented in Tinker? 
If this is the case, I think this would be a great addition to Tinker.

Thanks in advance!
- Jacek

**Comment by jayponder** [comment]
I think the link you have included above is not correct?

In any case, I believe Steven Boxer's group at Stanford did what you are describing roughly a decade ago or longer- ie, modified Tinker to apply an external field during MD simulation. That may be the work you are referencing in your comment? Their work was done a long time ago, and we do not have their code. In principle, I think this might not be too hard to implement, though I'd need to think through any possible issues with PME, periodic boundaries, etc.

**Reply by jacekkozuch** [reply]
Yes, sorry, that's the correct link (also corrected above):
https://aip.scitation.org/doi/full/10.1063/1.4861893
The authors were citing your paper in the sentence, in which their modified code was mentioned and I must have copied it instead.

I don't think anybody in Steven Boxer's lab has done that. I was a Postdoc in his lab and before me only Stephen Fried has used Tinker for AMOEBA MD simulations in 2013 (I think you may be referring to this paper: https://www.boxerlab.stanford.edu/_files/ugd/4006c9_5213e42b8e6d4b05b5f2b4bb55b4176c.pdf), but these were "only" solvation electrostatics , not external electric fields. To my knowledge the paper above by Gao et al. is the only one, in which external electric field were applied in Tinker. 

If it were possible to implement external electric fields, this would be something very interesting and useful in my opinion. And I would highly appreciate it! I've also contacted the corresponding author of the Gao et al. paper, but I haven't gotten any response yet.
